tools/scripts/script.ps1

```diff
@@ -0,0 +1,242 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+param(
+    [Parameter(HelpMessage = "Specify this parameter to uninstall AIShell")]
+    [switch] $Uninstall
+)
+
+<#################################################
+#
+#               Helper functions
+#
+#################################################>
+
+function Get-PowerShellVersion {
+    if ([version]$PSVersionTable.PSVersion -lt [version]"7.4.6") {
+        Write-Warning "[PowerShell version 7.4.6 is the minimum requirement for AIShell module, exiting...]"
```

Comment:
```suggestion
        Write-Error "PowerShell v7.4.6 is required for using the AIShell module."
```
I think we should add more content to the message about where to get v7.4.6 PowerShell. Maybe point to the release page  https://aka.ms/PowerShell-Release?tag=v7.4.6? /cc @StevenBucher98, thoughts?

Reply:
Agreed we should add where they can install PS 7.4.6